src/azure-cli/azure/cli/command_modules/network/_validators.py

```diff
@@ -1870,6 +1870,20 @@ def validate_status_code_ranges(namespace):
     namespace.status_code_ranges = values
 
 
+def validate_capture_size_and_limit(namespace):
+    if namespace.capture_limit:
+        if namespace.capture_limit < 0:
+            raise CLIError('usage error: --capture-limit cannot be a negative value.')
+
+    if namespace.capture_size:
+        if namespace.capture_size < 0:
+            raise CLIError('usage error: --capture-size cannot be a negative value.')
+
+    if namespace.time_limit:
+        if namespace.time_limit < 0:
+            raise CLIError('usage error: --time-limit cannot be a negative value.')
+
```

Comment:
Can we please add some unit tests for this ?

Reply:
Here the validation is only for negative values. I can add the UT in the next iteration with the change of support of vmss.